src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -654,6 +654,8 @@ internal SyntaxTree SyntaxTree
             }
         }
 
+        public sealed override Location TryGetFirstLocation() => Location;
```

Comment:
>public sealed override Location TryGetFirstLocation() => Location; [](http://example.com/codeflow?start=8&length=66)

Please place this override next to ```public override ImmutableArray<Location> Locations``` to make it obvious why implementation is correct.